swap/swap_test.go

```diff
@@ -1069,6 +1073,40 @@ func TestPeerProcessAndVerifyChequeInvalid(t *testing.T) {
 	}
 }
 
+func TestSwapLogToFile(t *testing.T) {
+
+	var err error
+
+	dir, err := ioutil.TempDir("", "swap_test_log")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(dir)
+
+	//Test log in fileSetSwaplogpath
+	debug.SetSwaplogpath(dir)
+
+	TestResetBalance(t)
+
+	files, err := ioutil.ReadDir(dir)
+	if err != nil || len(files) == 0 {
+		t.Fatal(err)
+	}
+
+	fileP := path.Join(dir, files[0].Name())
+
+	var b []byte
+	b, err = ioutil.ReadFile(fileP)
+	if err != nil {
+		t.Fatal(err)
+	}
+	s := string(b)
+	if !strings.Contains(s, "sending cheque") {
+		t.Fatalf("expected the log to contain sending cheque")
```

Comment:
be more explicit here:
`expected the log contain the phrase 'sending cheque'`
(or `sentence` instead of `phrase`, or equivalent)

Reply:
I'm not sure I understood the error message.
Example --> log does not contain swap message ...
Could this be a valid error

Reply:
use quotation marks to indicate the substring that you are looking for :smiley:

Reply:
fixed